contrib/freshmaker-cli

```diff
@@ -377,6 +453,10 @@ parser.add_argument(
 parser.add_argument(
     '--limit', default=10, type=int,
     help="Return only last N events")
+parser.add_argument('--days-ago', default=7, dest="days", type=int,
```

Comment:
Rather than validate the value in `get_advisories_from_errata`, you can just do the following. It would also guard against someone entering a value less than 1.
```suggestion
parser.add_argument('--days-ago', default=7, dest="days", type=int, choices=range(1, 8),
```